docs/Analyzer Configuration.md

```diff
@@ -229,29 +231,29 @@ Configurable Rules: [CA2241](https://docs.microsoft.com/dotnet/fundamentals/code
 Option Values: Names of additional string formatting methods (separated by '|').
 Allowed method name formats:
 
-  1. Method name only (includes all methods with the name, regardless of the containing type or namespace)
+  1. Method name only (includes all methods with the name, regardless of the containing type or namespace).
   2. Fully qualified names in the [symbol's documentation ID format](https://github.com/dotnet/csharplang/blob/master/spec/documentation-comments.md#id-string-format)
-     with an optional "M:" prefix.
+     with an optional `M:` prefix.
 
-Default Value: None
+Default Value: _None_
 
 Examples:
 
 | Option Value | Summary |
 | --- | --- |
-|`dotnet_code_quality.additional_string_formatting_methods = MyFormat` | Matches all methods named 'MyFormat' in the compilation
-|`dotnet_code_quality.additional_string_formatting_methods = MyFormat1\|MyFormat2` | Matches all methods named either 'MyFormat1' or 'MyFormat2' in the compilation
-|`dotnet_code_quality.additional_string_formatting_methods = NS.MyType.MyFormat(ParamType)` | Matches specific method 'MyFormat' with given fully qualified signature
-|`dotnet_code_quality.additional_string_formatting_methods = NS1.MyType1.MyFormat1(ParamType)\|NS2.MyType2.MyFormat2(ParamType)` | Matches specific methods 'MyFormat1' and 'MyFormat2' with respective fully qualified signature
+| `dotnet_code_quality.additional_string_formatting_methods = MyFormat` | Matches all methods named 'MyFormat' in the compilation. |
+| `dotnet_code_quality.additional_string_formatting_methods = MyFormat1\|MyFormat2` | Matches all methods named either 'MyFormat1' or 'MyFormat2' in the compilation. |
```

Comment:
Is this one of the places where it doesn't render correctly in VSCode?

Reply:
Yes, for GitHub it is rendered as `|` while it is rendered as `\|` for VS Code.

Reply:
Can you try something like the following and check if it renders correctly in both?

```md
<code>dotnet_code_quality.additional_string_formatting_methods = NS1.MyType1.MyFormat1(ParamType)&#124;NS2.MyType2.MyFormat2(ParamType)</code>
```

Reply:
MD0033 no inline html code :)